packages/code-du-travail-data/package.json

```diff
@@ -5,15 +5,16 @@
   "private": true,
   "main": "indexing/index.js",
   "scripts": {
-    "build": "ncc build ./indexing/index.js -o dist && yarn -s dump > ./dist/dump.data.json",
+    "build": "yarn build:populate && yarn dump-dev",
+    "build:populate": "ncc build ./indexing/index.js -o dist/populate",
+    "check-slugs": "node -r esm indexing/slug_checker",
+    "download-suggestion": "curl -L $SUGGEST_DATA_URL -o ./dataset/suggestions.txt",
+    "dump-dev": "node -r esm dump.js > ./dist/dump.data.json",
     "lint": "eslint ./indexing/*",
```

Comment:
im not sure building the populate is necessary, running with `-r esm` it not enough ?

Reply:
From what I understand, building populate allow to have a smaller docker image since we only copy the populate script in a nodejs image

Reply:
Ok will try when i'll have some free time (lol)